index.html

```diff
@@ -1075,12 +1091,16 @@ <h2 class="accordion__question">How can I stay updated with new blog posts?</h2>
     </div>
 </section>
 <!-- Word of the Day Section -->
-<div id="word-of-the-day" style="background-color: rgb(248, 181, 181); padding: 20px; border-radius: 5px; color: black;">
-    <h2>Word of the Day</h2>
-    <p id="word"></p>
-    <p id="definition"></p>
+<div id="wordContainer">
+    <div id="word">Word of the Day</div>
+    <div id="definition">Definition will appear here.</div>
+    <img id="image" src="" alt="Word Image">

```

Comment:
what exactly are you trying to do here ? there is no image in the src 
what do you wish to display ?

Reply:
![image](https://github.com/user-attachments/assets/531ef197-1471-4b08-94e2-e435971a31ba)
this is how it looks like , fix this

Reply:
Ok I will fix it